src/NHSD.BuyingCatalogue.Identity.Api/Controllers/AccountController.cs

```diff
@@ -99,6 +105,28 @@ public async Task<IActionResult> Login(LoginViewModel viewModel)
             throw new Exception("Invalid return URL");
         }
 
+        [HttpGet]
+        public async Task<IActionResult> Logout(string logoutId)
+        {
+            return await Logout(new LogoutViewModel
+            {
+                LogoutId = logoutId
+            });
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<IActionResult> Logout(LogoutViewModel logoutViewModel)
```

Comment:
Should we have an `Authorize` attribute here? While redirecting to login just to logout again may be pointless is it worth considering to prevent hitting logout unless a user is already logged in?

Reply:
The few samples I have seen on this have allowed anonymous access. I see this as attempting to delete a user that has already been deleted. Don't really need to tell the user that. They got what they wanted.